src/core_plugins/kibana/public/visualize/wizard/step_1.html

```diff
@@ -48,6 +48,7 @@ <h2 class="kuiSubTitle kuiVerticalRhythm">
           ng-href="{{ getVisTypeUrl(type) }}"
           tooltip="{{ getVisTypeTooltip(type) }}"
           tooltip-placement="{{ getVisTypeTooltipPosition($parent.$index) }}"
+          aria-describedby="visDescription_{{::type.name}}"
```

Comment:
Another dev wrote a function to normalize camelCase and other delimited text as screen readable. Might this be of use here?